hack/build-images.sh

```diff
@@ -15,6 +15,7 @@ STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/common.sh"
 source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/contrib/vagrant/provision-util.sh"
```

Comment:
Move os::util::install-sdn into its own file and put it int contrib/node, then reference it from vagrant.